src/Model/Translatable/TranslatablePropertiesTrait.php

```diff
@@ -5,18 +5,20 @@
 namespace Knp\DoctrineBehaviors\Model\Translatable;
 
 use Doctrine\Common\Collections\Collection;
-use Knp\DoctrineBehaviors\Contract\Entity\TranslationInterface;
 
+/**
+ * @template T of \Knp\DoctrineBehaviors\Contract\Entity\TranslationInterface
+ */
 trait TranslatablePropertiesTrait
 {
     /**
-     * @var Collection<string, TranslationInterface>
+     * @var Collection<string, T>
      */
     protected $translations;
```

Comment:
This breaks the PHPStorm autocomplete. 
Could you share the gif with verification it won't happen?

Thanks

Reply:
You're right, it's a little too early as template with restriction (`@template T of ..`) will be available in version 2022.3

Reply:
That is now working since PHPStorm introduced support for generics and this notation.